sw/device/tests/power_virus_systemtest.c

```diff
@@ -182,62 +184,6 @@ static uint32_t csrng_reseed_cmd_header;
  */
 static volatile uint32_t peripheral_clock_period_ns;
 
-/**
- * Pinmux pad configurations.
- */
-static const pinmux_pad_attributes_t pinmux_pad_attributes[] = {
```

Comment:
As we apply this settings only for the simulation, I moved them to the configure_pinmux_sim() below